sympy/physics/mechanics/particle.py

```diff
@@ -157,20 +162,22 @@ def angular_momentum(self, point, frame):
 
         """
 
-        return self.point.pos_from(point) ^ (self.mass * self.point.vel(frame))
+        return cross(self.point.pos_from(point),
+                     (self.mass * self.point.vel(frame)))
```

Comment:
Potential to remove unnecessary parentheses. I think it makes the indentation slightly easier to read.
```suggestion
        return cross(self.point.pos_from(point),
                     self.mass * self.point.vel(frame))
```